### PR TITLE
Init2 nw policy p4

### PR DESCRIPTION
**Pull request by dhruvpt**
New pull request, which is same as init-nw-policy-p4, but with resolved comments.

### CODE REVIEW

**Review by shweta503 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -381,12 +379,14 @@ Kubernetes is known to not work well with Linux swap and hence, it should be tur
...
-  ip link set P4TAP_0 name TAP_0
-  ip link set TAP_0 up
-  ip addr add 169.254.1.1/32 dev TAP_0
-  ARP_PROXY_IF=TAP_0 ./bin/arp_proxy &
+  ip netns add arpns
+  ip link set P4TAP_0 netns arpns
+  ip netns exec arpns ip link set P4TAP_0 up
+  ip netns exec arpns ip addr add 169.254.1.1/32 dev P4TAP_0
+  ip netns exec arpns ip route add default dev P4TAP_0 scope link
+  ip netns exec arpns ARP_PROXY_IF=P4TAP_0 ./bin/arp_proxy &
```
Comment (jokuniew) [inline README.md]: This line keeps telling me that `exec of "ARP_PROXY_IF=P4TAP_0" failed: No such file or directory`
When I do `ip netns exec arpns ls ./bin` I can see the arp_proxy binary

It used to work on previous version. Shouldn't be there any more changes done?

File: README.md [hunk 2]
```diff
@@ -371,7 +369,7 @@ Kubernetes is known to not work well with Linux swap and hence, it should be tur
 ## P4-K8s Deployment
   Run create_interfaces.sh script which, in addition to creating specified number of TAP interfaces, sets up the huge pages and starts P4-OVS.
   ```bash
-  # ./p4-k8s/scripts/create_interfaces.sh <8/16/32/...> [OVS_DEP_INSTALL_PATH]
+  # ./p4-k8s/scripts/create_interfaces.sh <8/16/32/...> $P4OVS_DEPS_INSTALL
```
Comment (jokuniew) [inline README.md:372]: Shouldn't it be `$OVS_INSTALL/P4OVS_DEPS_INSTALL`? 

in line 319 we had `$PWD/P4OVS_DEPS_INSTALL` and then `export OVS_INSTALL=$PWD`

File: README.md (hunk 1 above)
Comment (nupurjai) [inline README.md]: export ARP_PROXY_IF=P4TAP_0 && ./bin/arp_proxy &

Comment (jokuniew) [reply README.md]: In my dev-env I workaround this with `ip netns exec arpns bash -c "ARP_PROXY_IF=P4TAP_0 ./bin/arp_proxy &"`.
@dhruvpt  could you check that? c:

Comment (jokuniew) [reply README.md]: `ip netns exec arpns export ARP_PROXY_IF=P4TAP_0 && ./bin/arp_proxy &`
gives `exec of "export" failed: No such file or directory`

**Review by nupurjai — COMMENTED** [review]
Please do not add pb.bin for now. Need service to come up first with some changes

File: README.md (hunk 1 above)
Comment (jokuniew) [reply README.md]: @nupurjai

File: README.md (hunk 2 above)
Comment (dhruvpt) [reply README.md:372]: After sourcing the OVS env file, the $P4OVS_DEPS_INSTALL gets set to point to the same location.

File: README.md (hunk 1 above)
Comment (dhruvpt) [reply README.md]: @jokuniew I will look into it. For the time being, you can completely skip this line of running arp_proxy and it should still work.

**Comment by dhruvpt** [comment]
> Please do not add pb.bin for now. Need service to come up first with some changes

We can wait for the services to come up and then, merge this PR. Let me know when that happens.

**Pull request closed without merging**